<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UThControlOptions</class>
 <widget class="QWidget" name="UThControlOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="actGroupBox">
     <property name="title">
      <string>Activity (²³⁴U/²³⁸U) curves</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="actMinSpinBox">
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="actMaxSpinBox">
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="actStepSpinBox">
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="title">
      <string>Time lines</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="timeMinSpinBox">
          <property name="suffix">
           <string> ka</string>
          </property>
          <property name="maximum">
           <number>1500</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="timeMaxSpinBox">
          <property name="suffix">
           <string> ka</string>
          </property>
          <property name="maximum">
           <number>1500</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="timeStepSpinBox">
        <property name="suffix">
         <string> ka</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="detGroupBox">
     <property name="title">
      <string>Detrital ²³⁰Th correction</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="detModeComboBox">
        <item>
         <property name="text">
          <string>Assumed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Isochron</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QFrame" name="assumedFrame">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>²³⁰Th/²³²Th</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="assValueLineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>±</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="assUncertLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QFrame" name="isoFrame">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Fit type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="detFitComboBox">
           <item>
            <property name="text">
             <string>York</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OLS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Huber</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ageGroupBox">
     <property name="title">
      <string>Calculate age</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
